examples/kubernetes/dynamic_provisioning/README.md

```diff
@@ -19,7 +19,9 @@ parameters:
 * deploymentType (Optional) - FSx for Lustre supports three deployment types, SCRATCH_1, SCRATCH_2 and PERSISTENT_1. Default: SCRATCH_1.
 * kmsKeyId (Optional) - for deployment type PERSISTENT_1, customer can specify a KMS key to use.
 * perUnitStorageThroughput (Optional) - for deployment type PERSISTENT_1, customer can specify the storage throughput. Default: "200". Note that customer has to specify as a string here like "200" or "100" etc.
-
+* automaticBackupRetentionDays (Optional) - The number of days to retain automatic backups. The default is to retain backups for 7 days. Setting this value to 0 disables the creation of automatic backups. The maximum retention period for backups is 35 days
```

Comment:
Can you help add links to FSX new features and we can easily help review?

Reply:
https://aws.amazon.com/about-aws/whats-new/2020/06/amazon-fsx-lustre-provides-highly-durable-file-system-backups/

Reply:
https://docs.aws.amazon.com/fsx/latest/LustreGuide/using-backups-fsx.html